draft-lenders-dns-over-coap.md

```diff
@@ -119,17 +119,11 @@ The terms "CoAP payload" and "CoAP body" are used as defined in [RFC7959].
 Selection of a DoC Server
 =========================
 
-TBD:
-
-- URI or should we rather go straight to CRI?
-- Support for more than one URI by DoC server.
-- DoC server identity, key exchange, ...
-
-URI Alternatives
-----------------
-TBD:
-
-- CRI {{?I-D.ietf-core-href}} or CoRAL {{?I-D.ietf-core-coral}}
+It is assumed in this document that the DoC server, as well as the DNS query resource at the DoC
+server, is known to the DoC client. Possible ways to configure could be manual configuration with a
+URI {{?RFC3986}} or CRI {{?I-D.ietf-core-href}} or via a CoRE resource directory
+{{?I-D.ietf-core-resource-directory}}. Automatic configuration SHOULD only be done from a trusted
+source.
 
```

Comment:
I think the `rt=core.dns` is still something we'll need to mention (and later register) -- otherwise users get a hint but not the full set of necessary tools.

```suggestion

When discovering the URI through a link mechanism such as where the {{?RFC6690}} attribute "resource type" is available, the resource type TBDcore.dns can be used to identify a generic DNS resolver that is available to the client.
```

Reply:
I assume this also needs to be added to the IANA considerations then?

Reply:
Eventually, yes. (But a "TBD: register core.dns" can suffice in a pinch).